.github/workflows/unittest.yml

```diff
@@ -6,7 +6,7 @@ jobs:
       runs-on: ubuntu-20.04
```

Comment:
changes to this file should be generated with eg the `vsc.install.ci` tool. i propose to start with py3.6 and 3.9 only

Reply:
I agree, I'll make a ticket for vsc-install

Reply:
https://github.com/hpcugent/vsc-install/issues/214